pubspec.yaml

```diff
@@ -15,7 +15,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 4.1.0+1
+version: 4.2.0+1
```

Comment:
This isn't related to this PR, but @apomalyn  what is the +1 after the version? I keep wondering what it does

Reply:
aha let me bring light to these numbers :smile: this is the build number or build metadata ^^
If you want to read the complete version of SemVer head [here](https://semver.org/) :smile: if you need me to complete my answer don't hesitate to ask :smile:

Reply:
Ohh nice! Thanks for the info 😄